pkgs/development/coq-modules/coqtail-math/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, mkCoqDerivation, coq, version ? null }:
+
+with lib;
+
+mkCoqDerivation {
+  pname = "coqtail-math";
+  owner = "coq-community";
+  inherit version;
+  defaultVersion = if versions.range "8.10" "8.12" coq.coq-version then "20201124" else null;
```

```suggestion
  defaultVersion = if versions.range "8.11" "8.13" coq.coq-version then "20201124" else null;
```

This version of `coqtail-math` works with Coq 8.13 but does not work with Coq 8.10.